src/js/utils/behavior.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+const assign = require('object-assign');
+const forEach = require('array-foreach');
+const Behavior = require('receptor/behavior');
+
+const sequence = function () {
```

Comment:
But the `es2015` babel config you're using with `babelify` will take care of this :)